index.html

```diff
@@ -510,6 +519,55 @@ <h3>Security and Privacy Considerations</h3>
 
 		</section>
 
+		<section id="audio-ua-behaviour">
+			<h2>User Agent Behaviours for Audiobooks</h2>
+
+			<p>This section will outline the expected user agent behaviours for implementation of audiobooks. For processing instructions, user agents should refer to the <a href="https://www.w3.org/TR/pub-manifest/#manifest-processing"><code>Processing a Manifest</code></a> section of the Publication Manifest specification, and conform to any behaviour described there. This section will outline any additional user agent behaviours for audiobooks.</p>
+
+			<p>All user agent behaviours described in this section are intended to provide implementors with guidance, not strict requirements. Behaviours in this document will be mainly taken from the <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note published by the working group.</p>
+
```

Comment:
> will be taken mainly

->

> is taken mainly

Reply:
I'm going to be asking Franco and Josh to edit PWP UCR.